src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java

```diff
@@ -161,32 +163,36 @@ private boolean doSave() {
         }
     }
 
-    public boolean save() {
-        return save(SaveDatabaseMode.NORMAL);
+    public boolean save(BibDatabaseContext bibDatabaseContext) {
```

Comment:
Why is the database context passed as an method argument? The base panel is already an constructor argument, thus I would expect that the `SaveAction` always operates on this database. Maybe set the database context in a new field in the constructor?